frontend/src/concepts/pipelines/content/createRun/utils.ts

```diff
@@ -39,6 +39,7 @@ const runTypeSafeDates = (runType: RunFormData['runType']): boolean =>
 
 export const isFilledRunFormData = (formData: RunFormData): formData is SafeRunFormData =>
   !!formData.nameDesc.name &&
+  !!formData.experiment &&
```

Comment:
do you need experiment. wont it just default to `Default`. Do we want to pre-fill the field with default? Does have an opinion on this?